src/rdb.c

```diff
@@ -2211,7 +2211,9 @@ robj *rdbLoadObject(int rdbtype, rio *rdb, sds key, int dbid, int *error) {
             }
 
             if (rdbtype == RDB_TYPE_HASH_2 && itemexpiry > 0) {
-                hashTypeTrackEntry(o, entry);
+                // TODO xbasel: We need to add the hashtabele object serverDb->to keys_with_volatile_items
+                // the follow invocation will blow up, because o doesn't have the key yet, its created outside this function , seedbAddRDBLoad
+                // hashTypeTrackEntry(&server.db[dbid], o, entry);
```

Comment:
so what is the path forward here? how does the added fields being added to the internal object tracking?